src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Prepare an Azure subscription to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure subscription with approriate role definition and service principal for operating Giant Swarm guest clusters."
+date = "2018-08-09"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure subscription to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the host cluster. This gives great flexibility depending on the requirements and the use case. For example, it allows the host cluster to be running in an Azure subscription owned by Giant Swarm, while guest clusters operate in different Azure subscriptions each, depending on a customer’s department using them.
+Giant Swarm operates guest clusters using a service called `azure-operator` which runs on the host cluster.
+
+## Overview
+
+In order to run Giant Swarm guest clusters, an Azure subscription needs the following elements :
+
+- Role definition: a set of permission to operate guest clusters in the Azure subscription.
+- Service principal: an identity (bound to the previoulsy defined role definition) to access the Azure subscription.
+
+## Create Azure role definition and service principal
```

Comment:
Capitalisation for `role definition` and `service principal` needs to be consistent here